### ISSUE TITLE
Wrong warnings in tslint.json for accepted configuration values

### ISSUE THREAD

**Issue by Martin-Wegner** [palantir/tslint#1776]
I'm using VSCode 1.7.2, TSLint plugin 0.6.3, TSLint 4.0.1 and TypeScript 2.0.10.

When I open the tslint.json file in VSCode I get serveral wrong warnings about my configuration values:

`"cyclomatic-complexity": true,` -> `Incorrect type. Expected "array"`

`"no-consecutive-blank-lines": true,` -> `Incorrect type. Expected "array"`

`"object-literal-key-quotes": [true, "consistent-as-needed"],` -> `Value is not an accepted value. Valid values: [true,false,"always","as-needed"]`

`"one-variable-per-declaration": true,` -> `Incorrect type. Expected "array"`

`"only-arrow-functions": true,` -> `Incorrect type. Expected "array"`

`"typedef": [true, "arrow-call-signature"],` -> `Value is not an accepted value. Valid values: [true,false,"call-signature","parameter","arrow-parameter","property-declaration","variable-declaration","member-variable-declaration"]`

https://github.com/Microsoft/vscode-tslint/issues/132

**Comment by egamma** [comment]
VS Code exposes the errors as returned from the findConfiguration() call, pls see:
https://github.com/Microsoft/vscode-tslint/blob/master/tslint-server/src/server.ts#L236

**Comment by adidahiya** [comment]
Those don't look like errors thrown by TSLint, but rather JSON schema validation errors produced by validating against something like https://github.com/SchemaStore/schemastore/blob/54900f99ddc1870a17cc7c12492e815494deffd5/src/schemas/json/tslint.json

**Comment by Martin-Wegner** [comment]
@adidahiya should I open an issue there?

**Comment by egamma** [comment]
@adidahiya just to clarify, we are talking about the error property of the IConfigurationLoadResult.

`tslint\src\configuration.ts`
```typescript
export function findConfiguration(configFile: string, inputFilePath: string): IConfigurationLoadResult {
    const path = findConfigurationPath(configFile, inputFilePath);
    const loadResult: IConfigurationLoadResult = { path };

    try {
        loadResult.results = loadConfigurationFromPath(path);
    } catch (error) {
        loadResult.error = error;  // THIS IS THE ERROR OBJECT 
    }

    return loadResult;
}
```

**Comment by Martin-Wegner** [comment]
@egamma done https://github.com/SchemaStore/schemastore/issues/245
